pom.xml

```diff
@@ -1173,86 +1173,10 @@
-            <!-- Excluded due to Maven ANTLR4 Plugin's issue #978: https://github.com/antlr/antlr4/issues/978-->
-            <excludedLink>http://www.antlr.org/antlr4-maven-plugin</excludedLink>
-            <!-- till https://github.com/mojohaus/versions-maven-plugin/pull/49 -->
-            <excludedLink>http://mojo.codehaus.org/versions-maven-plugin</excludedLink>
-            <!-- Excluded due to Maven Release Plugin's issue #919: https://issues.apache.org/jira/browse/MRELEASE-919-->
-            <excludedLink>http://maven.apache.org/plugins/maven-release-plugin/</excludedLink>
-            <!-- Excluded till https://issues.sonatype.org/browse/NEXUS-9643 -->
-            <excludedLink>http://www.sonatype.com/public-parent/nexus-maven-plugins/nexus-staging/nexus-staging-maven-plugin/</excludedLink>
-            <!-- Excluded till https://github.com/mojohaus/build-helper-maven-plugin/issues/3 -->
-            <excludedLink>http://www.mojohaus.org/build-helper-maven-plugin/build-helper-maven-plugin</excludedLink>
-            <!-- This link exists, but it has too much content and github has problems with rendering it quickly, so to avoid ocasional "Read timed out" we exclude it -->
-            <excludedLink>https://github.com/checkstyle/checkstyle/issues/1555</excludedLink>
-            <!-- Excluded, because the link is generated by sonar's maven report plugin and we
-            can not correct the link while it will not be changed in effective pom.xml of the plugin
-            by sonar's developers. It is not allowed to open issue against sonar's maven report plugin
-            at sonar's issue tracker: http://jira.codehaus.org/browse/MSONAR
-            Sonar's mailing lists: http://www.mojohaus.org/sonar-maven-plugin/mail-lists.html-->
-            <excludedLink>https://sonarqube.com/project/index/com.puppycrawl.tools:checkstyle</excludedLink>
-            <!-- skip validating SNAPSHOTs on sourceforge when ${projectVersion} is SNAPSHOT-->
-            <excludedLink>http://downloads.sourceforge.net/project/checkstyle/checkstyle/*-SNAPSHOT/checkstyle-*-SNAPSHOT-all.jar</excludedLink>
-            <!-- skip validating released "-all" jar on sourceforge as it is uploaded after site generation-->
-            <excludedLink>http://downloads.sourceforge.net/project/checkstyle/checkstyle/*/checkstyle-*-all.jar</excludedLink>
-            <!-- Excluded due to linkcheck's issue #22: https://issues.apache.org/jira/browse/MLINKCHECK-22-->
-            <excludedLink>https://twitter.com/checkstyle_java/</excludedLink>
-            <!-- Excluded due to linkcheck's issue #23: https://issues.apache.org/jira/browse/MLINKCHECK-23-->
-            <excludedLink>http://www.kclee.com/clemens/java/javancss</excludedLink>
-            <!-- It is not a strict link, no needt ovalidate, github gives "429 Too Many Requests" for it -->
-            <excludedLink>https://github.com/search*</excludedLink>
```

Comment:
BTW I have rebased again and the codeship you have specified isn't building still.  Also travis seems to fail on one of the tests but I am not sure why since I am not even touching that part of the code.